tox.ini

```diff
@@ -44,15 +39,18 @@ deps =
 basepython =
     python2.7
 commands =
-    {[testing]covercmd} --cover-min-percentage=100
+    {[testing]covercmd}
+    coverage report --show-missing --fail-under=100
```

Comment:
@jonparrott This seems to be unnecessary, as the `--cov=` flags make `py.test` spit out the exact same table that `covage report ...` spits out.